content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -97,13 +97,11 @@ $ dvc stage add --name train \
                 --params seed,lr,weight_decay \
                 --checkpoints model.pt \
                 --plots-no-cache predictions.json \
-                --live dvclive \
+                --plots-no-cache dvclive/scalars \
```

Comment:
Any particular reason to use `--plots-no-cache` instead of `--plots` like above?

Reply:
I think that we should encourage `plots-no-cache` in the default example usages (for  `dvclive/scalars`)

Reply:
What do you mean as default? Do you mean this is an implicit recommendation? Well, up to you here @daavoo , I think no-cache makes sense for small text files Git can handle but maybe that context is lost in the example and regular `--plots` is def. easier to read.

> Maybe the Q is whether `--plots` should by default not cache the outputs but that's a core Q and I think it was discussed some time ago.